include/odp/api/spec/timer.h

```diff
@@ -36,16 +36,45 @@ extern "C" {
  */
 
 /**
- * Clock sources for timers in timer pool.
+ * Clock sources for timer pools
+ *
+ * ODP_CLOCK_DEFAULT is the default clock source and it is supported always. It is implementation
+ * defined which other clock sources are supported. See implementation documentation how the
+ * supported clock sources are mapped into these enumerations.
```

Comment:
Maybe better: `See from implementation documentation how the...` or `See implementation documentation for information on how the...`

Reply:
Fixed with "See from .." in v2